File: curso_poo_uber/java/CarBasic.java
package platzi.curso_poo_uber.java;

public class CarBasic extends Car{
    String brand;
    String model;
    
    public CarBasic(String licence, Account driver, String brand, String model) {
        super(licence, driver);
        this.brand = brand;
        this.model = model;
    }
    
}

File: curso_poo_uber/PHP/CarBasic.php
<?php
require_once('Car.php');
class CarBasic extends Car{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model) {
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    public function PrintDataCar(){
        echo " <br> brand: $this->brand <br> model: $this->model";
    }

}
?>